components/dashboard/src/settings/Plans.tsx

```diff
@@ -294,8 +339,8 @@ export default function () {
     } else {
         const targetPlan = freePlan;
         let bottomLabel;
-        if (scheduledDowngradePlanId === targetPlan.chargebeeId) {
-            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)">Cancel</a></p>;
+        if (scheduledDowngradePlanId === targetPlan.chargebeeId && !pendingDowngradeCancellation) {
+            bottomLabel = <p className="text-green-600">Downgrade scheduled<br/><a className="text-blue-light leading-6" href="javascript:void(0)" onClick={() => confirmCancelDowngrade()}>Cancel</a></p>;
```

Comment:
**issue**: Also, when you click the Cancel link the downgrade process is stuck in that state when you close the modal unless you do cancel downgrade, right?

Reply:
It shouldn't be stuck, but rather stay as is until the next billing cycle.

Reply:
Yes, it makes sense after https://github.com/gitpod-io/gitpod/pull/3873#discussion_r615992156. 🏁